### PR TITLE
Added Document-Topic Proportion in DTM tutorial.

### PR DESCRIPTION
**Pull request by bhargavvader**
@tmylk  Added a few lines on how to print Document-Topic proportions in the tutorial, something which would commonly be used but wasn't mentioned how to do before.

It's also not an explicit function and you have to do some digging in the wrapper code comments to figure out how to do so, so I thought this would be useful.

### CODE REVIEW

**Comment by tmylk** [comment]
Thanks!

File: docs/notebooks/dtm_example.ipynb [hunk 1]
```diff
@@ -182,38 +199,77 @@
...
+      "Distribution of Topic 0 0.706412\n",
+      "Distribution of Topic 1 0.293588\n"
+     ]
+    }
+   ],
+   "source": [
+    "doc_number = 1\n",
+    "num_topics = 2\n",
+    "\n",
+    "for i in range(0,num_topics):\n",
```
Comment (piskvorky) [inline docs/notebooks/dtm_example.ipynb]: PEP8: space after comaa.

File: docs/notebooks/dtm_example.ipynb [hunk 2]
```diff
@@ -182,38 +199,77 @@
...
+    "    print \"Distribution of Topic %d %f\" % (i,model.gamma_[doc_number,i])"
```
Comment (piskvorky) [inline docs/notebooks/dtm_example.ipynb]: PEP8 again: spaces.

Comment (bhargavvader) [inline docs/notebooks/dtm_example.ipynb]: Created new PR to address this.

**Comment by piskvorky** [comment]
@tmylk did you review this before merging?

**Pull request merged**